src/Reveal.Sdk.Dom.Tests/Visualizations/CandleStickVisualizationFixture.cs

```diff
@@ -0,0 +1,362 @@
+using System.Collections.Generic;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using Reveal.Sdk.Dom.Core.Serialization;
+using Reveal.Sdk.Dom.Data;
+using Reveal.Sdk.Dom.Filters;
+using Reveal.Sdk.Dom.Visualizations;
+using Xunit;
+
+namespace Reveal.Sdk.Dom.Tests.Visualizations;
+
+public class CandleStickVisualizationFixture
+{
+    [Fact]
+    public void Constructor_InitializesDefaultValues_WhenInstanceIsCreated()
+    {
+        // Act
+        var candleStickVisualization = new CandleStickVisualization();
+
+        // Assert
+        Assert.NotNull(candleStickVisualization);
```

Comment:
Remove this. As the codes below already ensure that candleStickVisualization is not null